.editorconfig

```diff
@@ -0,0 +1,50 @@
+# editorconfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Default settings:
+# A newline ending every file
+# Use 4 spaces as indentation
+[*]
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+
+# License header
+file_header_template = Licensed to the .NET Foundation under one or more agreements.\nThe .NET Foundation licenses this file to you under the MIT license.
```

Comment:
I was expecting the CI to fail for some System.Buffers files that have the wrong license header.

I know this fails in roslyn-analyzers, but what are we missing?https://github.com/dotnet/roslyn-analyzers/blob/b924542a1b526322929725a1aaa9586c21b1b231/.editorconfig#L176C3-L176C3